tests/package-lock.json

```diff
@@ -1,8 +1,6006 @@
 {
   "name": "ts-tests",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
did this file have to be updated to fix the tests? most of the diff for this PR is now this file

Reply:
Oh interesting. I don't think this needs to be updated. @joelamouche What are the consequences of this. Do we want the package-lock committed?

Reply:
Reverted the changes and the CI is still green. If those package-lock changes are desirable we can have them in another PR.